Index: src/com/google/javascript/jscomp/CompilationLevel.java
===================================================================
--- src/com/google/javascript/jscomp/CompilationLevel.java	(revision 2416)
+++ src/com/google/javascript/jscomp/CompilationLevel.java	(working copy)
@@ -72,6 +72,8 @@
     options.removeClosureAsserts = false;
     // Don't shadow variables as it is too confusing.
     options.shadowVariables = false;
+    options.setRenamingPolicy(
+        VariableRenamingPolicy.OFF, PropertyRenamingPolicy.OFF);
   }
 
   /**
